tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/settings/JmapSettings.scala

```diff
@@ -48,7 +49,23 @@ case class JmapSettingsValue(value: String) extends AnyVal
 
 trait JmapSettingEntry
 
-case class JmapSettings(settings: Map[JmapSettingsKey, JmapSettingsValue], state: UuidState)
+object JmapSettings {
+  val trashCleanupEnabledSetting: JmapSettingsKey = JmapSettingsKey.liftOrThrow("trash.cleanup.enabled")
```

Comment:
IMO, `TRASH_CLEANUP_ENABLED_SETTING` upper snake cake for the static variable will help us  easy read whenever invoke

Reply:
According to https://docs.scala-lang.org/style/naming-conventions.html, it should be TrashCleanupEnabledSetting :))